schema-sandbox/schema.json

```
@@ -326,8 +326,10 @@
                 "languages": {
                     "type": "array",
                     "items": {
-                        "type": "string",
-                        "maxLength": 3
+                        "format": "^[a-z]{2,3}$",
```

```suggestion
                        "pattern": "^[a-z]{2,3}$",
```

Should be `pattern` = regex pattern.